docs/index.rst

```diff
@@ -81,6 +81,8 @@ The Python bindings also provide some additional configuration properties:
 * ``error_cb(kafka.KafkaError)``: Callback for generic/global error events. This callback is served by
   poll().
 
+* ``stats_cb(json_str)``: Callback for statistics data. This callback is triggered by poll() every ``statistics.interval.ms`` (needs to be configured separately). Function argument ``json_str`` is a str instance of a JSON document containing statistics data.
```

Comment:
docs are good, but split up the lines at about 80 columns so it is readable in the terminal too.